engine-tests/src/tests/one_inch.rs

```diff
@@ -40,7 +39,7 @@ fn test_1inch_liquidity_protocol() {
     let (result, profile, pool) =
         helper.create_pool(&pool_factory, token_a.0.address, token_b.0.address);
     assert!(result.gas_used >= 4_500_000); // more than 4.5M EVM gas used
-    assert!(profile.all_gas() <= 103_000_000_000_000); // less than 103 NEAR Tgas used
+    assert!(profile.all_gas() <= 120_000_000_000_000); // less than 103 NEAR Tgas used
```

Comment:
103 Tgas -> 120 Tgas is a pretty big change. Going to 104 or 105 might be ok, but if this code reorganization (ostensibly a non-functional change) is now consuming 16% more gas then I'm a little worried.

Do we have any sense why splitting things out into separate crates is having a performance impact?

Reply:
Fixed in https://github.com/aurora-is-near/aurora-engine/pull/243/commits/def784c5fd9b2cc733c2c5033c3e3f5fffe4d3e0